apps/designer/app/designer/features/sidebar-left/panels/components/components.tsx

```diff
@@ -83,6 +83,9 @@ const elementToComponentName = (element: Element) => {
     return;
   }
   const { dragComponent } = parentWithData.dataset;
+  const componentNames = (
+    Object.keys(components) as Array<Instance["component"]>
+  ).filter((component) => components[component].isListed);
```

Comment:
I moved these lines to the top of the file. Need to update them there to use `isListed`. And probably no point of having them here now.